Sources/MapboxNavigation/CarPlaySearchControllerDelegate.swift

```diff
@@ -41,4 +42,15 @@ public protocol CarPlaySearchControllerDelegate: AnyObject {
      animated or not.
      */
     func popTemplate(animated: Bool)
+    
+    var recentItems: [RecentItem] { get set }
```

Comment:
We must update these newly added protocol symbols.

Reply:
With doc comments?

Reply:
Yep, it doesn't have to happen in this PR though. It'd greatly improve readability.